### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-misc/official-ios-remote-main/) for Kodi misc/Official iOS Remote (main).

Current translation status:

Weblate translation status

### CODE REVIEW

File: XBMC Remote/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -151,15 +151,15 @@
 "Audiostream not available" = "音频流不可用";
 "Audiostreams not available" = "音频流不可用";
 "LongDateTimeFormat" = "YYYY年M月d日";
-"ShortDateTimeFormat" = "yyyy-MM-dd";
+"ShortDateTimeFormat" = "年年年年-月月-日日";
```
Comment (kambala-decapitator) [inline XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: @gade01 is there a way to prevent such changes?

Comment (gade01) [reply XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: @kambala-decapitator There's really nothing we can do about it.
Basically translators can write anything.
We can make a comment on the string at Weblate: https://kodi.weblate.cloud/translate/kodi-misc/official-ios-remote-main/zh_cn/?offset=1&q=%E5%B9%B4%E5%B9%B4%E5%B9%B4%E5%B9%B4-%E6%9C%88%E6%9C%88-%E6%97%A5%E6%97%A5&sort_by=-priority%2Cposition&checksum=

Comment (kambala-decapitator) [reply XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: thanks.

After checking the code I think we'd better get rid of these strings entirely, will open an issue.

Comment (gade01) [reply XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: That's probably a good idea.

Comment (kambala-decapitator) [reply XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: the correct way would be merging this PR first and deleting strings afterwards, right?

Comment (gade01) [reply XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: Yeah, that will avoid merge conflicts at Weblate.

Comment (gade01) [reply XBMC Remote/zh-Hans.lproj/Localizable.strings:154]: And you actually only need to remove en_gb strings.
Weblate should take care of the rest.

**Comment by wutschel** [comment]
What do you think about the incomplete translation of the "Show" options to switch off/on the different main menu entires? Wouldn't it look better to not show any incomplete translation and stick to the English version until all strings are translated?

**Comment by gade01** [comment]
@wutschel 
>What do you think about the incomplete translation of the "Show" options to switch off/on the different main menu entires? Wouldn't it look better to not show any incomplete translation and stick to the English version until all strings are translated?

Well, that's how translations are. 
I'm fine with it as it i now and most other translations for other addons and Kodi core have incomplete translations like this.
But it's totally up to you.

**Comment by wutschel** [comment]
> Well, that's how translations are. 

Ok, let us move on and merge this. Afterwards we can merge https://github.com/xbmc/Official-Kodi-Remote-iOS/pull/449 which removes the unwanted `"ShortDateTimeFormat"` and `"LongDateTimeFormat"` strings and applies another translation fix.

**Review by gade01 — APPROVED** [review]

**Pull request merged**